webpack.config.js

```diff
@@ -51,6 +51,7 @@ module.exports = {
     new webpack.ProvidePlugin({
       'fetch': 'imports?this=>global!exports?global.fetch!whatwg-fetch'
     }),
+    new webpack.ContextReplacementPlugin(/buffer/, require('buffer')),
```

Comment:
There is a bug in Buffer module inside webpack, so what's this doing is making sure `buffer` version that we specified in package.json is used.